### DISCUSSION TITLE
Como sair do comando git log?

### DISCUSSION THREAD

**Discussion by maria-lourenco** [digitalinnovationone/dio-lab-open-source#34398] [Git Bash]
Olá! Quando fiz o commit ele buscou as informacoes de todos que contribuiram no projeto ,porem tem um que nao me permite continuar .

https://github.com/digitalinnovationone/dio-lab-open-source/assets/168197247/fe664f27-4b3c-4bcf-9091-bc0a016d6f73

**Comment by ReisAglemar** [comment]
Parece que você digitou um ' git log'. esse comando faz uma spécie de stream de dados, para sair desse stream use a tecla 'q', para descer na stream use 'd' e para subuir use 'u'  q = quit; d = down ;u = up

**Reply by maria-lourenco** [reply]
muito obrigada , teve uma pessoa no discord que me orientou a fazer isso mesmo e deu certo ,salvei sua explicação para não esquecer os outros comandos .Agora estou com dificuldade em conectar o vscode com github.

**Reply by ReisAglemar** [reply]
https://www.youtube.com/watch?v=peGUkhXD3Vw

veja esse tutorial, link acima, penso que vai resolver o seu problema.

**Comment by maria-lourenco** [comment]
ok agradeco muito pela ajuda ,vou fazer conforme o tutorial.


Em qui., 16 de mai. de 2024 às 15:26, Reis ***@***.***>
escreveu:
